src/masternodes/rpc_proposals.cpp

```diff
@@ -0,0 +1,601 @@
+
+#include <masternodes/mn_rpc.h>
+
+#include <functional>
+
+UniValue propToJSON(CPropId const& propId, CPropObject const& prop)
+{
+    UniValue ret(UniValue::VOBJ);
+    ret.pushKV("proposalId", propId.GetHex());
+    ret.pushKV("title", prop.title);
+    ret.pushKV("type", CPropTypeToString(prop.type));
+    ret.pushKV("status", CPropStatusToString(prop.status));
+    ret.pushKV("amount", ValueFromAmount(prop.nAmount));
+    ret.pushKV("cyclesPaid", int(prop.cycle));
+    ret.pushKV("totalCycles", int(prop.nCycles));
+    ret.pushKV("finalizeAfter", int64_t(prop.finalHeight));
+    ret.pushKV("payoutAddress", ScriptToString(prop.address));
+    return ret;
+}
+
+UniValue propVoteToJSON(CPropId const& propId, uint8_t cycle, uint256 const & mnId, CPropVoteType vote)
+{
+    UniValue ret(UniValue::VOBJ);
+    ret.pushKV("proposalId", propId.GetHex());
+    ret.pushKV("masternodeId", mnId.GetHex());
+    ret.pushKV("cycle", int(cycle));
+    ret.pushKV("vote", CPropVoteToString(vote));
+    return ret;
+}
+
+/*
+ *  Issued by: any
+*/
+UniValue createcfp(const JSONRPCRequest& request)
+{
+    auto pwallet = GetWallet(request);
+
+    RPCHelpMan{"createcfp",
+               "\nCreates a Cummunity Fund Request" +
```

Comment:
Typo, should be "Community". Also createcfp should be createcfr as it is a community fund request. Pinkpaper refers to the call as creategovcfr so uses cfr as well. I think we can take some liberty with the actual call name, but if we use cfp then the help should be renamed Community Fund Proposal.

Reply:
Pinkpaper should be updated to show correct nick name `createcfp`, i will update the help, it should be `Community Fund Proposal`.